openedx/core/djangoapps/user_api/accounts/views.py

```diff
@@ -569,24 +583,72 @@ def retirement_partner_report(self, request):  # pylint: disable=unused-argument
             is_being_processed=False
         ).order_by('id')
 
-        retirements = [
-            {
-                'user_id': retirement.user.pk,
-                'original_username': retirement.original_username,
-                'original_email': retirement.original_email,
-                'original_name': retirement.original_name,
-                'orgs': self._get_orgs_for_user(retirement.user),
-                'created': retirement.created,
-            }
-            for retirement in retirement_statuses
-        ]
+        retirements = []
+        for retirement_status in retirement_statuses:
+            retirements.append(self._get_retirement_for_partner_report(retirement_status))
 
         serializer = UserRetirementPartnerReportSerializer(retirements, many=True)
 
         retirement_statuses.update(is_being_processed=True)
 
         return Response(serializer.data)
 
+    def _get_retirement_for_partner_report(self, retirement_status):
+        """
+        Get the retirement for this retirement_status. The retirement info will be included in the partner report.
+        """
+        retirement = {
+            'user_id': retirement_status.user.pk,
+            'original_username': retirement_status.original_username,
+            AccountRetirementPartnerReportView.ORIGINAL_EMAIL_KEY: retirement_status.original_email,
+            AccountRetirementPartnerReportView.ORIGINAL_NAME_KEY: retirement_status.original_name,
+            'orgs': self._get_orgs_for_user(retirement_status.user),
+            'created': retirement_status.created,
+        }
+
+        # Some orgs have a custom list of headings and content for the partner report. Add this, if applicable.
+        self._add_orgs_config_for_user(retirement, retirement_status.user)
+
+        return retirement
+
+    def _add_orgs_config_for_user(self, retirement, user):
+        """
+        Check to see if the user's info was sent to any partners (orgs) that have a a custom list of headings and
+        content for the partner report. If so, add this.
+        """
+        # See if the MicroBachelors coaching provider needs to be notified of this user's retirement
+        if has_ever_consented_to_coaching is not None and has_ever_consented_to_coaching(user):
```

Comment:
Can `has_ever_consented_to_coaching is not None and has_ever_consented_to_coaching(user)` just be `if has_ever_consented_to_coaching and has_ever_consented_to_coaching(user)`?